docker/api/Dockerfile

```
@@ -1,6 +1,6 @@
 ARG VERSION=unspecified
 
-FROM python:3.10.0-alpine
+FROM python:3.10.1
```

Latest version of Python 3.10 and we should specify the underlying OS to avoid poor assumptions.

```suggestion
FROM 3.10.2-slim-bullseye
```